### DISCUSSION TITLE
Guide - Migration from v1 to v2x

### DISCUSSION THREAD

**Discussion by kriss145** [medusajs/medusa#9196] [Q&A] [answered]
Hi,

I very like Medusa.js and how it works. Can't wait for 2.x version.

Is anywhere guide where is description how to migrate Medusa.js from version 1.x to version 2.x or will be create?

I can't find any information about it.

**Comment by olivermrbl** [comment]
We will publish a migration guide shortly after the official release of 2.0 :)

**Reply by joekendal** [reply]
Hi, how long is it until official release? Are we talking a few weeks or a few months?

**Reply by harshjv** [reply, marked as answer]
@joekendal https://github.com/medusajs/medusa/discussions/6689#discussioncomment-10840593

**Comment by kosyak** [comment]
Greetings,

is migration guide still planned? It’d be very unfortunate if we couldn’t keep orders, customers etc after the upgrade or won’t have a clean upgrade path.

**Reply by alessioacella** [reply]
up

**Reply by stephanboersma** [reply]
Up

**Comment by alessioacella** [comment]
@olivermrbl any news on this?
